### ISSUE TITLE
Adds extraneous parentheses around array index

### ISSUE THREAD

**Issue by lrowe** [jlongster/prettier#173]
As of 0.0.6 array indices have extraneous parentheses added. Not a problem with 0.0.5.

Input:
```js
var x = y[0];
```
Output:
```js
var x = y[(0)];
```

**Comment by jlongster** [comment]
This is definitely caused by https://github.com/jlongster/prettier/pull/141. We'll get a fix out asap, as this is a pretty critical bug.

**Comment by jlongster** [comment]
Also I can't believe none of the flow tests has basic array lookup syntax. I'm hoping we just messed up the snapshots somehow or something.

**Comment by vjeux** [comment]
There are 251 lines that have `[0]` in them fyi

```
grep -R '\[0\]' tests | wc -l
     251
```

**Comment by vjeux** [comment]
Fixed by #179
